### PR TITLE
feat(calling): show double tap toast in an active call

### PR DESCRIPTION
**Pull request by ohassine**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
    - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
    - [x] contains a reference JIRA issue number like `SQPIT-764`
    - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
    - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

### Description

- Show a one time display time toast to indicate to user that he can double tap to go full screen.
- Show a toast in full screen to indicate to user that he should double tap to go back(will be visible every time we open that screen)

### Attachments (Optional)

https://github.com/wireapp/wire-android-reloaded/assets/18124919/1fb0f5c1-ecfe-49be-8614-123e7f2802bc

----
#### PR Post Submission Checklist for internal contributors (Optional)

- [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

- [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-1778/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-1778/2/) **failed**.

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-1778/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6483805)(6483805)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6483805)(6483805)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-1778/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6487515)(6487515)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6487515)(6487515)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [5](https://10.10.124.134/job/wire-android-reloaded/job/PR-1778/5/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6517941)(6517941)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6517941)(6517941)-staging-compat.apk)

**Comment by AndroidBob** [comment]
Build [6](https://10.10.124.134/job/wire-android-reloaded/job/PR-1778/6/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6518656)(6518656)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6518656)(6518656)-staging-compat.apk)

**Review by alexandreferris — APPROVED** [review]

**Review by saleniuk — APPROVED** [review]

File: app/src/main/kotlin/com/wire/android/ui/calling/ongoing/fullscreen/FullScreenTile.kt [hunk 1]
```diff
@@ -68,6 +79,21 @@ fun FullScreenTile(
                 onSelfUserVideoPreviewCreated = sharedCallingViewModel::setVideoPreview,
                 onClearSelfUserVideoPreview = sharedCallingViewModel::clearVideoPreview
             )
+            LaunchedEffect(Unit) {
+                delay(200)
```
Comment (saleniuk) [inline app/src/main/kotlin/com/wire/android/ui/calling/ongoing/fullscreen/FullScreenTile.kt:83]: maybe extract this also to the companion object?

**Review by gongracr — APPROVED** [review]
LGTM 👨‍🍳 🥘

**Comment by AndroidBob** [comment]
Build [7](https://10.10.124.134/job/wire-android-reloaded/job/PR-1778/7/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv4.2.0(6537204)(6537204)-staging-compat.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/staging/compat/com.wire.android-vv4.2.0(6537204)(6537204)-staging-compat.apk)

**Review by trOnk12 — APPROVED** [review]

**Comment by trOnk12** [comment]
NIce one, maybe it would be good idea to bring the team to the atention that we have this component in our arsenal 🤔

**Review by yamilmedina — APPROVED** [review]

File: app/src/main/kotlin/com/wire/android/ui/calling/ongoing/fullscreen/FullScreenTile.kt (hunk 1 above)
Comment (ohassine) [reply app/src/main/kotlin/com/wire/android/ui/calling/ongoing/fullscreen/FullScreenTile.kt:83]: It's not visible on screen launch that's why it is different

**Pull request merged**